x-pack/solutions/security/plugins/security_solution/public/management/pages/trusted_apps/view/components/form.test.tsx

```diff
@@ -316,6 +316,36 @@ describe('Trusted apps form', () => {
       expect(screen.getByText(INPUT_ERRORS.pathWarning(0))).not.toBeNull();
     });
 
+    it('should show path malformed path warning for linux/mac without an executable name', () => {
+      render();
+      expect(screen.queryByText(INPUT_ERRORS.pathWarning(0))).toBeNull();
+      expect(screen.queryByText(INPUT_ERRORS.wildcardPathWarning(0))).toBeNull();
+
+      const propsItem: Partial<ArtifactFormComponentProps['item']> = {
+        os_types: [OperatingSystem.LINUX],
+        entries: [createEntry(ConditionEntryField.PATH, 'match', '/')],
+      };
+      formProps.item = { ...formProps.item, ...propsItem };
+      render();
+      expect(screen.getByText(INPUT_ERRORS.pathWarning(0))).not.toBeNull();
+      expect(screen.queryByText(INPUT_ERRORS.wildcardPathWarning(0))).toBeNull();
+    });
+
+    it('should show path malformed path warning for windows with no executable name', () => {
+      render();
+      expect(screen.queryByText(INPUT_ERRORS.pathWarning(0))).toBeNull();
+      expect(screen.queryByText(INPUT_ERRORS.wildcardPathWarning(0))).toBeNull();
+
+      const propsItem: Partial<ArtifactFormComponentProps['item']> = {
+        os_types: [OperatingSystem.WINDOWS],
+        entries: [createEntry(ConditionEntryField.PATH, 'match', 'c:\\fold\\')],
+      };
+      formProps.item = { ...formProps.item, ...propsItem };
+      render();
+      expect(screen.getByText(INPUT_ERRORS.pathWarning(0))).not.toBeNull();
+      expect(screen.queryByText(INPUT_ERRORS.wildcardPathWarning(0))).toBeNull();
+    });
```

Comment:
I was thinking about these tests a bit more and I believe it would be nice to group all these tests for better readability within `is` and `matches` groups. `describe ('is operator')` (that would contain all tests with `match`) and `describe ('matches operator') (which would contain all tests that use `wildcard` operator). We can also further group tests within these into `mac| linux`/`windows`  groups.

That would help in the future to make changes again. Reading the tests quickly would tell us what's the intended behaviour and that would help in the future if someone is not sure about the UX.